include/constants.h

```diff
@@ -75,6 +76,7 @@ namespace reinforcement_learning {  namespace value {
       const char *const CONSOLE_TRACE_LOGGER = "CONSOLE_TRACE_LOGGER";
       const char *const NULL_TIME_PROVIDER = "NULL_TIME_PROVIDER";
       const char *const CLOCK_TIME_PROVIDER = "CLOCK_TIME_PROVIDER";
+      const char *const DECISION_RANK_MODE = "DEFAULT";
```

Comment:
`DEFAULT` is very vague. Can you find a self descriptive name ?

Reply:
I talked about it with Ed. Using "Online" to indicate the default state at this moment.